Added a missing word to "Prevent Cross-Site Scripting (XSS) in ASP.NET Core" article (#29641)
dotnet/AspNetCore.Docs: aspnetcore/security/cross-site-scripting.md

## Accessing encoders in code

> The HTML, JavaScript and URL encoders are available to your code in two ways, you can inject them via [dependency injection](xref:fundamentals/dependency-injection) or you can use the default encoders contained in the `System.Text.Encodings.Web` namespace. If you use the default encoders, then any customizations you applied to character ranges to be treated as safe won't take effect - the default encoders use the safest encoding rules possible.

Reviewer: ```suggestion
The HTML, JavaScript and URL encoders are available to your code in two ways:

* Inject them via [dependency injection](xref:fundamentals/dependency-injection).
* Use the default encoders contained in the `System.Text.Encodings.Web` namespace.

When using the default encoders, then any customizations applied to character ranges to be treated as safe won't take effect. The default encoders use the safest encoding rules possible.
```
